frontend/src/stimulus/controllers/dynamic/filter/filters-form.controller.ts

```diff
@@ -318,26 +316,20 @@ export default class FiltersFormController extends Controller {
   }
 
   sendForm() {
-    const params = new URLSearchParams();
-    params.append('filters', this.buildFiltersParam(this.parseFilters()));
+    const params = new URLSearchParams(window.location.search);
```

Comment:
The `page` param would need to be excluded as otherwise that is kept between filter sets. So when the user is on e.g. the second page and then changes the filters, the results of the second page are displayed (if available) whereas the user should be presented the results on the first page.